lib/stdlib/doc/src/maps.xml

```diff
@@ -364,6 +364,26 @@ none</code>
       </desc>
     </func>
 
+    <func>
+      <name name="merge_deep" arity="2" since=""/>
+      <fsummary></fsummary>
+      <desc>
+        <p>Merges two maps, and if two keys exist in both maps
```

Comment:
By "two keys exist" do you mean "the same key exists"?

Reply:
thank you, sounds better.

Reply:
fixed